app/src/main/java/net/gsantner/opoc/util/StringUtils.java

```diff
@@ -195,4 +195,50 @@ public static boolean isNewLine(CharSequence source, int start, int end) {
         Collections.addAll(list, array);
         return list;
     }
+
+    /**
+     * Convert escape sequences in string to escaped special characters
+     *
+     * For example, convert
+     *
+     * A\tB -> A    B
+     *
+     * A\nB -> A
+     *         B
+     *
+     * @param input Input string
+     * @return String with escaped sequences converted
+     * @throws IllegalArgumentException if an invalid escape sequence is found in the string
+     */
+    public static String unescapeString(final String input) throws IllegalArgumentException {
```

Comment:
This code allows a user to use special characters such as newline `\n` and tab `\t` in the replace text box